FireRender.Maya.Src/Context/FireRenderContext.cpp

```diff
@@ -424,25 +424,30 @@ bool FireRenderContext::buildScene(bool isViewport, bool glViewport, bool freshe
 	return true;
 }
 
+static const std::vector<int> denoiser_aovs = { 
```

Comment:
1. Please rename variable if tis intended to be global
2. Should we move it to class scope ?